src/Components/Input.tsx

```diff
@@ -124,7 +124,7 @@ class Input extends React.Component<InputProps, InputState> {
       )
     }
 
-    return <StyledInput {...this.props} />
+    return <StyledInput {...this.props as any} />
```

Comment:
This change was also part of your TS bump PR, was that maybe unintentional?

Reply:
Yup, I rebased on top for the time being and have a note at the top of the PR.

Reply:
Ah kk